.github/pull_request_template.md

```diff
@@ -8,8 +8,13 @@ Describe the changes you're proposing.
 
 ## Technical writer support
 
-Do you need a tech writer's support, for example to review your PR?  
+Do you need a tech writer's support, for example to review your PR?
 
 ## How to review
 
 Tell reviewers how to assess your changes.
+
+## Confirm
+
+- [ ] I have checked if any docs change here also requires updates to other repositories (ADRs / RFCs, README.md, Team Manual, Elsewhere in these docs)
+- [ ] Where there is an overlap I have updated or opened a PR for corresponding changes
```

Comment:
maybe replace "an" with "any" just in case there are multiple overlaps we aren't making it sound like just one single overlap if there could be a few